puppet/modules/kubernetes/manifests/install.pp

```diff
@@ -19,4 +22,18 @@
     owner  => 'root',
     group  => 'root',
   }
+
+  exec {'sysctl-system':
+    command     => 'sysctl --system',
+    refreshonly => true,
+    path        => ['/usr/bin/', '/bin', '/usr/sbin'],
+  }
+
+  file{"${::kubernetes::params::sysctl_dir}/fs.conf":
+    ensure  => file,
+    mode    => '0640',
+    owner   => 'root',
+    content => template('kubernetes/fs.conf.erb'),
+    notify  => Exec['sysctl-system'],
+  }
```

Comment:
Can we do that as part of `::kubelet` instead of `::install`. 

Kubelet seems to be the part causing the issues with it's cadvisor integration.